spec/core_functions/color/in_gamut

```diff
@@ -0,0 +1,358 @@
+================================================================================
+<===> rgb/input.scss
+@use "sass:color";
+a {b: color.is-in-gamut(rgb(255 -1 0))}
+
+<===> rgb/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> rgba/input.scss
+@use "sass:color";
+a {b: color.is-in-gamut(rgba(300 122 180 / .2))}
+
+<===> rgba/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> rgba/input.scss
+@use "sass:color";
+a {b: color.is-in-gamut(rgba(-1 122 180 / .2))}
+
+<===> rgba/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> hwb/input.scss
+@use "sass:color";
+a {b: color.is-in-gamut(hwb(0 300% 0%))}
+
+<===> hwb/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> hwb/input.scss
+@use "sass:color";
+a {b: color.is-in-gamut(hwb(0 50% -1%))}
+
+<===> hwb/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> hsl/input.scss
+@use "sass:color";
+a {b: color.is-in-gamut(hsl(0 300% 0%))}
+
+<===> hsl/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> hsl/input.scss
+@use "sass:color";
+a {b: color.is-in-gamut(hsl(0 -1% 0%))}
+
+<===> hsl/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> srgb/input.scss
+@use "sass:color";
+a {b: color.is-in-gamut(color(srgb 2 0.07843 0.823530))}
+
+<===> srgb/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> srgb/input.scss
+@use "sass:color";
+a {b: color.is-in-gamut(color(srgb -1 0.07843 0.823530))}
+
+<===> srgb/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> srgb-linear/input.scss
+@use "sass:color";
+a {b: color.is-in-gamut(color(srgb-linear 2 0.07843 0.823530))}
+
+<===> srgb-linear/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> srgb-linear/input.scss
+@use "sass:color";
+a {b: color.is-in-gamut(color(srgb-linear -1 0.07843 0.823530))}
+
+<===> srgb-linear/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> display-p3/input.scss
+@use "sass:color";
+a {b: color.is-in-gamut(color(display-p3 2 0.35 0.3 / 1))}
+
+<===> display-p3/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> display-p3/input.scss
+@use "sass:color";
+a {b: color.is-in-gamut(color(display-p3 -1 0.35 0.3 / 1))}
```

Comment:
I'm not clear on what the division (`/`) is for in some of the color functions here? I think the `out-of-gamut` colors would still be formatted like other colors. We would just have find actually `out-of-gamut` colors to test.